R/data.R

```diff
@@ -0,0 +1,12 @@
+#' Late Summer Profile
+#'
+#' \describe{
+#'   \item{depth}{depth}
+#'   \item{temper}{temperature}
+#'   \item{salinity}{salinity}
+#'   \item{oxygen}{oxygen}
+#'   \item{oxygen.sat}{oxygen saturation}
+#'   \item{density}{Density}
```

Comment:
Could you add units and maybe a little description to this? It would be great to know what lake and region this is from. Whatever it is, it looks pretty deep!

Reply:
It is [Quesnel Lake](https://en.wikipedia.org/wiki/Quesnel_Lake) and yes she is a deep one! How does this look for documentation: https://github.com/GLEON/rLakeAnalyzer/pull/85/commits/9d55c55e6e2fed4886f7f74aa7cb8bb57d008a8f ?